community/lucene-index/src/main/java/org/neo4j/kernel/api/impl/index/IndexWriterConfigs.java

```diff
@@ -36,6 +36,10 @@
 {
     private static final int MAX_BUFFERED_DOCS =
             FeatureToggles.getInteger( IndexWriterConfigs.class, "max_buffered_docs", 100000 );
+    private static final int POPULATION_MAX_BUFFERED_DOCS =
+            FeatureToggles.getInteger( IndexWriterConfigs.class, "population_max_buffered_docs", IndexWriterConfig.DISABLE_AUTO_FLUSH );
```

Comment:
Maybe use `.` instead of `_` to be consistent with other feature toggle names in this class?